core/primitives/res/runtime_configs/364.txt

```diff
@@ -0,0 +1,2 @@
+wasm_ed25519_verify_base: 3_365_369_625_000s
+wasm_ed25519_verify_byte: 291_580_479
```

Comment:
Why does this file exists? I think it should be deleted. Adding to base parameter files alone is enough.
(Version 364 is not loaded anyway. If it was, you would notice the typo at the end of line 1.)

Reply:
removed